apps/web/app/(app)/[emailAccountId]/automation/onboarding/page.tsx

```diff
@@ -65,13 +65,40 @@ async function getUserPreferences({
   if (!emailAccount) return undefined;
 
   return {
-    toReply: getToReplySetting(emailAccount.rules),
-    coldEmails: getColdEmailSetting(emailAccount.coldEmailBlocker),
-    newsletter: getRuleSetting(SystemType.NEWSLETTER, emailAccount.rules),
-    marketing: getRuleSetting(SystemType.MARKETING, emailAccount.rules),
-    calendar: getRuleSetting(SystemType.CALENDAR, emailAccount.rules),
-    receipt: getRuleSetting(SystemType.RECEIPT, emailAccount.rules),
-    notification: getRuleSetting(SystemType.NOTIFICATION, emailAccount.rules),
+    toReply: {
+      action: getToReplySetting(emailAccount.rules) || "label",
+      digest: false,
```

Comment:
We would want to get the actual user preference here. Default would always be false if user doesn't have it set.